### PR TITLE
[IMP] l10n_at: taxes

### PR DESCRIPTION
**Pull request by malb-odoo**
We want to invigorate a Tax Name Taxonomy so that tax names are as codified as possible. This allow for a better display on Invoice Description, and allowed us to implement a smart name_search.

In this PR, we change the taxes name so that it's more clear for the users

task-id: 3052677

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_at/data/template/account.tax-at.csv [hunk 1]
```diff
@@ -1,219 +1,219 @@
...
-"account_tax_template_purchase_eu_xx_vst_076","VST_076 IGE (im Bestimmungsland besteuert)","","VSt. 20%","500","purchase","20.0","percent","False","False","","base","invoice","-KZ 076||-KZ 077","",""
-"","","","","","","","","","","","tax","invoice","","chart_at_template_2505",""
-"","","","","","","","","","","","base","refund","+KZ 076||+KZ 077","",""
-"","","","","","","","","","","","tax","refund","","chart_at_template_2505",""
-"account_tax_template_purchase_eu_xx_vst_077","VST_077 IGE (im Inland besteuert)","","VSt. 20%","500","purchase","20.0","percent","False","False","","base","invoice","","",""
-"","","","","","","","","","","","tax","invoice","","chart_at_template_2505",""
-"","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","tax","refund","","chart_at_template_2505",""
+"id","name","description","invoice_label","sequence","type_tax_use","amount","amount_type","price_include","active","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent",description@de
```
Comment (epictete) [inline addons/l10n_at/data/template/account.tax-at.csv:1]:
```suggestion
"id","name","description","invoice_label","sequence","type_tax_use","amount","amount_type","price_include","active","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@de"

```

File: addons/l10n_at/data/template/account.tax-at.csv [hunk 2]
```diff
@@ -1,219 +1,219 @@
...
+"account_tax_template_sales_rev_charge_0_code021",0% Ust L,UST_021 Tax liability concerns service recipient,"USt. 0%","400","sale","0.0","percent","False","False","","base","invoice","+KZ 021","","",UST_021 Steuerschuld betrifft Leistungsempfänger
```
Comment (epictete) [inline addons/l10n_at/data/template/account.tax-at.csv:2]:
```suggestion
"account_tax_template_sales_rev_charge_0_code021","0% Ust L","UST_021 Tax liability concerns service recipient","USt. 0%","400","sale","0.0","percent","False","False","","base","invoice","+KZ 021","","","UST_021 Steuerschuld betrifft Leistungsempfänger"

```
Several occurrences :wink:

**Review by epictete — COMMENTED** [review]
@robodoo r+

**Comment by Levizar** [comment]
=============== There are duplicates name with the same tax type ===============
20% Ust         : [38, 66]
10% Ust Add     : [58, 78]
20% Vst         : [156, 216]

**Pull request closed without merging**